includes/Admin/Editor.php

```diff
@@ -324,6 +324,8 @@ public function get_editor_settings(): array {
 					'hotlink'       => '/web-stories/v1/hotlink/',
 					'link'          => '/web-stories/v1/link/',
 					'statusCheck'   => '/web-stories/v1/status-check/',
+					'taxonomies'    => '/wp/v2/taxonomies/',
+					'taxonomyBase'  => '/wp/v2/',
```

Comment:
Why was this changed back? Should be `wp/v1/taxonomies`

Reply:
I'm getting 404s for everything else we've tried in here

Reply:
I'll try v1 again rq to double check

Reply:
<img width="762" alt="Screen Shot 2021-09-15 at 1 17 05 PM" src="https://user-images.githubusercontent.com/35983235/133487653-b84509e5-9f36-4989-ae48-4270f8845fc1.png">
<img width="766" alt="Screen Shot 2021-09-15 at 1 17 10 PM" src="https://user-images.githubusercontent.com/35983235/133487659-7393358f-160f-4eaf-8949-ef27e5cb9b7c.png">

Reply:
@littlemilkstudio You might need to run `composer install` first so the PHP classes are updated.

If that doesn't help, please ping me on Slack.